File: CompSci 1/Unit 18, Arrays/WordList/WordList.java
import java.util.Scanner;

public class WordList
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        String[] words = new String[10];

        System.out.println("Enter 10 words: ");
        for(int i = 0; i < 10; i++)
        {
            words[i] = input.next();
        }

        for(String s : words)
        {
            System.out.println(s);
        }
    }
}

File: CompSci 1/Unit 18, Arrays/WordList/WordList.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val input = Scanner(System.`in`)
    val words = arrayOfNulls<String>(10)

    println("Enter 10 words: ")
    for (i in 0..9) {
        words[i] = input.next()
    }

    for (s in words) {
        println(s)
    }
}